src/tools/projects.md

```diff
@@ -10,12 +10,14 @@ Below are a number of trailheads that lead into the projects and code that compr
 | [`moztelemetry`][mt]               | Scala APIs for Mozilla Telemetry
 | [`spark-hyperloglog`][spark_hll]   | Algebird's HyperLogLog support for Apache Spark
 | [`mozanalysis`][ma]                | A library for Mozilla experiments analysis
+| [`Glean`][glean]                   | A client-side telemetry SDK for collecting metrics and sending them to Mozilla's telemetry service.
```

Comment:
```suggestion
| [`glean`][glean]                   | A client-side mobile Telemetry SDK for collecting metrics and sending them to Mozilla's Telemetry service.
```

Seeing `Glean` I _have to_ send [the Spanish inquisition](https://www.youtube.com/watch?v=Ixgc_FGam3s) ... Mozlando decided it's: glean